bot/code_review_bot/cli.py

```diff
@@ -176,6 +179,19 @@ def main():
         # Update Harbormaster status
         w.update_status(revision, state=BuildState.Fail)
```

Comment:
I don't know if this is still necessary after we add `w.phabricator.update_build_target(`, maybe not

Reply:
I don't think we need it.